config/express.js

```diff
@@ -33,6 +33,14 @@ module.exports = function(db) {
 		require(path.resolve(modelPath));
 	});
 
+	// Globbing controllers files
+	config.getGlobbedFiles('./app/controllers/*.js').forEach(function(controllerPath) {
+		var controller = require(path.resolve(controllerPath));
+		if('init' in controller) {
```

Comment:
Why pass in `app`? Can you show a case where it is necessary? Why not also just do `if (controller.init)`?